integrations/googledrive/src/files-cache.ts

```diff
@@ -0,0 +1,86 @@
+import { RuntimeError, z } from '@botpress/sdk'
+import { baseGenericFileSchema } from './schemas'
+import { BaseFolderFile, BaseGenericFile, BaseNormalFile } from './types'
+import * as bp from '.botpress'
+
+const filesMapImpl = z.record(z.string(), baseGenericFileSchema)
```

Comment:
**question:** what does `Impl` mean ? Implementation ? Why not `filesMapSchema` ?

Reply:
Forgot to rename after refactoring